cfpb/consumerfinance.gov
# Rental assistance finder: fetch program JSON from s3

Updates RAF tool to fetch program JSON from s3


## Additions

- Fetch RAF JSON from s3
- Sort geographic results by city, then county, then state


## How to test this PR

1. Check out branch
2.  Run gulp copy
3. Navigate to [rental assistance tool page](http://localhost:8000/coronavirus/mortgage-and-housing-assistance/renter-protections/find-help-with-rent-and-utilities/)
4. In network tab of dev tools, verify that raf.json is fetched
5. Verify that tool loads and works as expected
6. Verify that filtered state results are displayed alphabetically in city / county / state order


## Screenshots
### Sorted results
![localhost_3000_ (1)](https://user-images.githubusercontent.com/778171/128393814-91ba4ccd-1be9-4cc6-bcd1-915e87d89ca6.png)


## Notes and todos

- We are in the process of publishing the RAF tool as an npm module, so this is the last PR that will use this method of importing the tool's JS.


### Front-end testing

#### Browser testing

Visually tested in the following supported browsers:
